content/en/news/releases/1.25.x/announcing-1.25/change-notes/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Istio 1.25.0 Change Notes
+linktitle: 1.25.0
+subtitle: Major Release
+description: Istio 1.25.0 release notes.
+publishdate: 2024-11-07
+release: 1.25.0
+weight: 10
+aliases:
+    - /news/announcing-1.25.0
+---
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/docs/releases/feature-stages/#feature-phase-definition). Please consider upgrading your environment to remove the deprecated functionality.
+
+- **Deprecated** use of `ISTIO_META_DNS_AUTO_ALLOCATE` in proxyMetadata in favor of https://istio.io/latest/docs/ops/configuration/traffic-management/dns-proxy/#dns-auto-allocation-v2. New users of Istio IP auto-allocation should adopt the new status-based controller. Existing users may continue to use the older implementation.
+  ([Issue #53596](https://github.com/istio/istio/issues/53596))
+
+## Traffic Management
+
+- **Promoted** the `cni.ambient.dnsCapture` value to default to `true`.
+This enables the DNS proxying for workloads in ambient mesh by default, improving security, performance, and enabling
+a number of features. This can be disabled explicitly or with `compatibilityVersion=1.24`.
+Note: only new pods will have DNS enabled. To enable for existing pods, pods must be manually restarted, or the iptables reconcilation feature must be enabled with `--set cni.ambient.reconcileIptablesOnStartup=false`.
+
+- **Promoted** the `PILOT_ENABLE_IP_AUTOALLOCATE` value to default to `true`.
+This enables the new iteration of [IP auto-allocation](/docs/ops/configuration/traffic-management/dns-proxy/#address-auto-allocation),
+fixing long-standing issues around allocation instability, ambient support, and increased visibility.
+`ServiceEntry` objects without `spec.address` set will now see a new field, `status.addresses`, automatically set.
+Note these will not be used unless proxies are configured to do DNS proxying, which remains off-by-default.
```

Comment:
"**Promoted** the `cni.ambient.dnsCapture` value to default to `true`" means that for ztunnel dnsCaprure will work out of the box so the "off-by-default" only applies to envoy-based proxies.